public/js/icinga/behavior/navigation.js

```diff
@@ -6,14 +6,22 @@
 
     Icinga.Behaviors = Icinga.Behaviors || {};
 
+    try {
+        var d3 = require("icinga/icinga-php-thirdparty/mbostock/d3");
+    } catch (e) {
+        console.warn('[Navigation] requires d3.js library');
+    }
+
     var Navigation = function (icinga) {
         Icinga.EventListener.call(this, icinga);
         this.on('click', '#menu a', this.linkClicked, this);
         this.on('click', '#menu tr[href]', this.linkClicked, this);
         this.on('rendered', '#menu', this.onRendered, this);
-        this.on('mouseenter', '#menu .primary-nav .nav-level-1 > .nav-item', this.showFlyoutMenu, this);
+        this.on('mousemove', '#menu .primary-nav .nav-level-1 > .nav-item', this.onMouseMove, this);
+        this.on('mouseenter', '#menu .primary-nav .nav-level-1 > .nav-item', this.onMouseEnter, this);
         this.on('mouseleave', '#menu .primary-nav', this.hideFlyoutMenu, this);
         this.on('click', '#toggle-sidebar', this.toggleSidebar, this);
+        this.on('resize', '', this.toggleConfigFlyout, this);
```

Comment:
?

Reply:
I think, I was trying to trigger an event on flyout resize and just left it as it was unnecessary.  This should be cleaned up.